Casks/e/epoccam.rb

```diff
@@ -9,11 +9,12 @@
 
   livecheck do
     url "https://help.elgato.com/api/v2/help_center/en-us/articles/360052826852.json"
-    strategy :page_match do |page|
-      match = page[/EpocCam[._-]Installer[._-]v?(\d+(?:[._]\d+)+)\.pkg/i, 1]
+    regex(/EpocCam[._-]Installer[._-]v?(\d+(?:[._]\d+)+)\.pkg/)
+    strategy :json do |json, regex|
+      match = json.dig("article", "body")&.match(regex)
       next if match.blank?
 
-      match.tr("_", ".")
+      match[1]&.tr("_", ".")
```

Comment:
```suggestion
      match[1].tr("_", ".")
```

With `String#match`, the return value will either be a `MatchData` object or `nil`. If `match` is a `MatchData` object (checked by `next if match.blank?`), then we know that any non-optional capture groups are available (i.e., `match[1]` should always be a non-`nil` value at this point). [Optional capture groups can be `nil` but we account for those in the `strategy` block logic.]

Reply:
I will keep that in mind, thank you!